### ISSUE TITLE
Use of HemholtzMedia in openmodelica 3.2.3

### ISSUE THREAD

**Issue by branch171** [thorade/HelmholtzMedia#36]
Hi, I have downloaded the latest version from GITHUB and loaded the library into OMEdit using 3.2.3. However, none of the examples for tests run. I think I am missing something. If I use ButaneTestModel_ph I get 

Check of HelmholtzMedia.Examples.MediaTestModels.ButaneTestModel_ph completed successfully.

Instantiation of HelmholtzMedia.Examples.MediaTestModels.ButaneTestModel_ph completed successfully.

But running fails, the first error is :

Internal error No support of solving not real variables with a non-linear solver. 

 Equation:
HelmholtzMedia.Examples.MediaTestModels.ButaneTestModel_ph.Medium.SaturationProperties(fixedMassFlowRate.medium.sat.psat, fixedMassFlowRate.medium.sat.Tsat, HelmholtzMedia.Examples.MediaTestModels.ButaneTestModel_ph.Medium.ThermodynamicState(fixedMassFlowRate.medium.sat.liq.phase, fixedMassFlowRate.medium.sat.liq.T, fixedMassFlowRate.medium.sat.liq.p, fixedMassFlowRate.medium.sat.liq.d, fixedMassFlowRate.medium.sat.liq.u, fixedMassFlowRate.medium.sat.liq.h, fixedMassFlowRate.medium.sat.liq.s), HelmholtzMedia.Examples.MediaTestModels.ButaneTestModel_ph.Medium.ThermodynamicState(fixedMassFlowRate.medium.sat.vap.phase, fixedMassFlowRate.medium.sat.vap.T, fixedMassFlowRate.medium.sat.vap.p, fixedMassFlowRate.medium.sat.vap.d, fixedMassFlowRate.medium.sat.vap.u, fixedMassFlowRate.medium.sat.vap.h, fixedMassFlowRate.medium.sat.vap.s)) = HelmholtzMedia.Examples.MediaTestModels.ButaneTestModel_ph.Medium.setSat_p(volume.medium.p) solve for {fixedMassFlowRate.medium.sat.psat,fixedMassFlowRate.medium.sat.liq.h,fixedMassFlowRate.medium.sat.vap.d,fixedMassFlowRate.medium.sat.vap.u,fixedMassFlowRate.medium.sat.vap.h,fixedMassFlowRate.medium.sat.vap.s,fixedMassFlowRate.medium.sat.liq.u,fixedMassFlowRate.medium.sat.vap.phase,fixedMassFlowRate.medium.sat.liq.s,fixedMassFlowRate.medium.sat.vap.T,fixedMassFlowRate.medium.sat.liq.phase,fixedMassFlowRate.medium.sat.liq.T,fixedMassFlowRate.medium.sat.Tsat,fixedMassFlowRate.medium.sat.vap.p,fixedMassFlowRate.medium.sat.liq.p,fixedMassFlowRate.medium.sat.liq.d}

Followed by 

Internal error createEquationsForSystems failed then many more 

Can you help get me going. Thanks

**Comment by thorade** [comment]
HelmholtzMedia is not yet fully supported by OpenModelica, see the current status in https://github.com/thorade/HelmholtzMedia/issues/34#issuecomment-623008764.
Please read that carefully and follow all links (especially the link to Library report)

You mention version 3.2.3, but that is the version of the Modelica Standard Library MSL.
What version of OpenModelica are you using?

![image](https://user-images.githubusercontent.com/314571/82566204-db416480-9b7b-11ea-823c-734a5ce64ff9.png)

**Comment by branch171** [comment]
Yes I have the same:

Connected to OpenModelica v1.14.1 (64-bit)
Connected to OMSimulator v2.1.0-dev-147-g36ec2c7-mingw

What software would need to run HelmholtzMedia

**Comment by thorade** [comment]
Sorry for the confusion: I used that screenshot above only to show where to find the version. Significant improvements have been made on OpenModelica side since 14.1 was released, so you should install the latest `Stable Development` or `Nightly Build`.
https://openmodelica.org/download/download-windows
This is especially convenient on Linux.
With that version, more than half of the examples should work; you can see what to expect here:
https://libraries.openmodelica.org/branches/newInst/HelmholtzMedia/HelmholtzMedia.html
or in https://github.com/thorade/HelmholtzMedia/issues/34#issuecomment-623008764

**Comment by branch171** [comment]
Hi, I was reading your paper "Partial derivatives of thermodynamic state properties for
dynamic simulation" and was wondering if you have ever came across the same analysis but for multiphase mixtures i.e. more than one component, so not pure. I have started to look at it. Most people do a numerical derivative but an analytical one would be more efficient for dynamic simulation. I found the multicomponent Clapeyron equation but was wondering if you have seen this attempted elsewhere.
